### PR TITLE
Fixed logic for dealing with eyekore OPHTHA classifications

### PR DESCRIPTION
**Pull request by aldum1**

### CODE REVIEW

File: OPHTHA_classifier.py [hunk 1]
```diff
@@ -91,6 +93,14 @@ def is_OCT_OPT(description):
...
     return common_utils.regex_search_label(regexes, description)
 
+# # Modality, OCT-OT
+# def is_OCT_OT(description):
+#     regexes = [
+#         #for Eyecor - Start with OT_
+#         re.compile('^OT_', re.IGNORECASE)
+#     ]
+#     return common_utils.regex_search_label(regexes, description)
+
```
Comment (NPann) [inline OPHTHA_classifier.py:103]: remove?

File: OPHTHA_classifier.py [hunk 2]
```diff
@@ -239,31 +259,33 @@ def classify_OPHTHA(dcm_metadata, acquisition):
             elif protocolName == 'Widefield OCT':
                 modality = 'OCT'
                 octType = ['Standard'] 
-                octSubType = 'Wide Field'
+                # octSubType = 'Wide Field'
```
Comment (NPann) [inline OPHTHA_classifier.py:262]: Remove ?

**Review by NPann — APPROVED** [review]
LGTM. Can we just remove the few commented out code blocks?

File: OPHTHA_classifier.py (hunk 1 above)
Comment (aldum1) [reply OPHTHA_classifier.py:103]: No, THis most likely will be enabled in the future

File: OPHTHA_classifier.py (hunk 2 above)
Comment (aldum1) [reply OPHTHA_classifier.py:262]: No, this will be enabled in the future

File: OPHTHA_classifier.py [hunk 3]
```diff
@@ -212,9 +227,13 @@ def classify_OPHTHA(dcm_metadata, acquisition):
                 modality = 'FP'
                 modalityType = 'Autofluorescence'
             elif protocolName in ['FP','FP-2','FP-3M','FP-4W','FP-7M','FP-7Std','FP-9','FP-ROP']:
-                modality == 'FP'
-                modalityType == 'Color'
-                subType = 'Standard'  
+                log.debug("Inside logic for ColorFundus using IN [FP, FP-2, and so on]")
+                modality = 'FP'
+                modalityType = 'Color' 
+                if protocolName == 'FP-4W':
```
Comment (aldum1) [inline OPHTHA_classifier.py:233]: Modified logic for Color Fundus type of field determination.
"Wide field" if protocolName is FP-4W else standard

**Review by aldum1 — COMMENTED** [review]
Modified logic for Color Fundus type of field determination.
Wide field if protocolName is 'FP-4W' else standard

**Pull request merged**